src/test/java/seedu/address/logic/commands/task/DeleteTaskCommandTest.java

```diff
@@ -0,0 +1,54 @@
+package seedu.address.logic.commands.task;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.commands.NewCommandTestUtil.VALID_ROOM_INDEX_ONE;
+import static seedu.address.logic.commands.NewCommandTestUtil.VALID_ROOM_INDEX_TWO;
+import static seedu.address.logic.commands.NewCommandTestUtil.VALID_TASK_INDEX_ONE;
+import static seedu.address.logic.commands.NewCommandTestUtil.VALID_TASK_INDEX_TWO;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+public class DeleteTaskCommandTest {
+
+    @Test
+    public void constructor_nullRoomIndexValidTaskIndex_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new DeleteTaskCommand(null, VALID_TASK_INDEX_ONE));
+    }
+
+    @Test
+    public void constructor_validRoomIndexNullTaskIndex_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new DeleteTaskCommand(VALID_ROOM_INDEX_ONE, null));
+    }
+
```

Comment:
Maybe add a success case for test?

Reply:
Seems like success cases are covered in equals() method test.

Reply:
It is missing the success case actually. Will need to set up a stub to perform the test.